src/apiPort.js

```diff
@@ -75,6 +77,20 @@ class ApiPort {
       })
     }
 
+    if (process.env.COMMON_DATA_CONFIG) {
```

Comment:
This change has broken the CI build(check https://travis-ci.org/testbetter/openapitest/builds/580571263?utm_source=github_status&utm_medium=notification), Also this should be committed on another branch, once it is not related to the open PR is function checked and allowed after if got error #15

Reply:
sry I missed that. Fixed :)